stored_queries/ncbi_tax/ncbi_taxon_search_sci_name.yaml

```diff
@@ -0,0 +1,21 @@
+# Search for an ncbi taxon with a scientific name
+# Results are limited to 10k
```

Comment:
Doesn't looks like they are limited to 10k? There probably should be limits and offset and limit though, things can get real bad based on personal experience if either of those are too large

Reply:
Changed this comment and added offset limit of 10k